docs/patterns/single-new-eks-observability-accelerators/single-new-eks-gpu-opensource-observability.md

```diff
@@ -0,0 +1,315 @@
+
+Run update-kubeconfig command. You should be able to get the command from CDK output message.
+
+```bash
+aws eks update-kubeconfig --name single-new-eks-opensource-observability-accelerator --region <your region> --role-arn arn:aws:iam::xxxxxxxxx:role/single-new-eks-opensource-singleneweksgpuopensourc...
+```
+
+Let’s verify the resources created by steps above:
+
+```bash
+kubectl get pods -A
+```
+
+Output:
+
+![GPU_Pods](../images/gpu/gpu_pods.png)
+
+Next, let's verify that each node has allocatable GPUs:
+
+```bash
+kubectl get nodes  "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"
+```
+
+Output:
+
+![GPU_List](../images/gpu/gpu_list.png)
+
+We can now deploy the [`nvidia-smi` binary](https://developer.download.nvidia.com/compute/DCGM/docs/nvidia-smi-367.38.pdf), which shows diagnostic information about all GPUs visible to the container:
+
+```bash
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nvidia-smi
+spec:
+  restartPolicy: OnFailure
+  containers:
+  - name: nvidia-smi
+    image: "nvidia/cuda:11.0.3-base-ubuntu20.04"
+    args:
+    - "nvidia-smi"
+    resources:
+      limits:
+        nvidia.com/gpu: 1
+EOF
+```
+
+Then request the logs from the Pod:
+
+```bash
+kubectl logs nvidia-smi
+```
+
+Output:
+
+![GPU_List](../images/gpu/gpu_nvidia_smi.png)
+
+## Visualization
+
+### Grafana NVIDIA DCGM Exporter Dashboard
+
+Login to your Grafana workspace and navigate to the Dashboards panel. You should see a dashboard named `NVIDIA DCGM Exporter Dashboard`.
+
+We will now generate some load, to see some metrics in the dashboard. Please run the following command from terminal:
+
+```bash
+cat << EOF | kubectl create -f -
+ apiVersion: v1
+ kind: Pod
+ metadata:
+   name: dcgmproftester
+ spec:
+   restartPolicy: OnFailure
+   containers:
+   - name: dcgmproftester11
+     image: nvidia/samples:dcgmproftester-2.0.10-cuda11.0-ubuntu18.04
+     args: ["--no-dcgm-validation", "-t 1004", "-d 120"]
+     resources:
+       limits:
+          nvidia.com/gpu: 1
+     securityContext:
+       capabilities:
+          add: ["SYS_ADMIN"]
+EOF
+```
+
+To verify the Pod was successfully deployed, please run:
+
+```bash
+kubectl get pods
+```
+
+Expected output:
+
+![GPU_dcgmproftester](../images/gpu/gpu_dcgmproftester.png)
+
+After a few minutes, looking into the `NVIDIA DCGM Exporter Dashboard`, you should see the gathered metrics, similar to: 
+
+![GPU_dcgm_Dashboard_1](../images/gpu/gpu_dcgm_1.png)
+
+![GPU_dcgm_Dashboard_2](../images/gpu/gpu_dcgm_2.png)
+
+![GPU_dcgm_Dashboard_3](../images/gpu/gpu_dcgm_3.png)
+
+Grafana Operator and Flux always work together to synchronize your dashboards with Git. If you delete your dashboards by accident, they will be re-provisioned automatically.
+    
+## Teardown
+
+You can teardown the whole CDK stack with the following command:
+
+```bash
+make pattern single-new-eks-gpu-opensource-observability destroy
+```
+
+## Troubleshooting
```

Comment:
This section is repeatative and can be removed.

Reply:
done